### DISCUSSION TITLE
Linux version size big compared to windows and os x?

### DISCUSSION THREAD

**Discussion by kkarthik23** [concourse/concourse#6576] [General]
Hi,
I believe a few others have mentioned that the  concourse server linux version is several times larger than the windows and os x versions

Is there a linux version which is similar in size to windows and mac versions?

if so where do we download it.

Appreciate your response.

Thanks

**Comment by taylorsilva** [comment]
Hey, the linux version is larger because it contains the images of all the base resource types (git, s3, time, etc.). Each of those images is a few 100MB's. It results in a better UX for people getting started with Concourse, but the binary is really large :(

The team does have a goal down the road to make a concourse "lite" for linux which would only come with the `registry-image` resource and it would be up to either:
1) operators to add their desired default resource types to concourse workers (you can currently do this already, add default resource types to workers)
2) users need to add resource types manually to all their pipelines. also something users can do today.

**Reply by kkarthik23** [reply]
thanks for the reply
